samples/drivers/kscan/src/main.c

```diff
@@ -15,7 +15,7 @@
 
 LOG_MODULE_REGISTER(main);
 
-#define KSCAN_NODE DT_ALIAS(kscan0)
+#define KSCAN_NODE DT_CHOSEN(zephyr_keyboard_scan)
```

Comment:
can we delete this extra definition? ie `DEVICE_DT_GET(DT_CHOSEN(zephyr_keyboard_scan));`

Reply:
done